packages/core/src/memdb.ts

```diff
@@ -66,9 +66,13 @@ class MemDb {
 
   async findAll<T extends Doc>(_class: Ref<Class<T>>, query: DocumentQuery<T>): Promise<T[]> {
     let result: Doc[]
-    if (query._id !== undefined) {
-      const obj = this.objectById.get(query._id)
-      result = obj !== undefined ? [obj] : []
+    if (Object.prototype.hasOwnProperty.call(query, '_id')) {
```

Comment:
Why query._id not work?

Reply:
do not work for queries like { _id: undefined }

Reply:
consider I have a query `{ _id: currentApp}`, so when `currentApp === undefined`, I think this query should return nothing. It was returning all the apps without this fix.

Reply:
Ok.